tools/wpt/run.py

```diff
@@ -474,6 +486,9 @@ def setup_logging(kwargs, default_config=None):
         default_config = {default_formatter: sys.stdout}
     wptrunner.setup_logging(kwargs, default_config)
     logger = wptrunner.logger
+    has_log = HasLogHandler()
+    logger.add_handler(mozlog.handlers.LogLevelFilter(has_log, "CRITICAL"))
+    return has_log
```

Comment:
@jgraham in what other way would you like to get the reference to the handler?

Reply:
I think all this code should be moved into wptrunner itself, since run.py is a frontend and isn't used in all browser integrations that are using wptrunner (e.g. gecko only uses it for non-Firefox runs).